### ISSUE TITLE
Control for playback speed

### ISSUE THREAD

**Issue by serpro69** [jjmaldonis/obsidian-audio-notes#3]
Hello again. 
Would  be nice to be able to control playback speed of embedded audio.

**Comment by jjmaldonis** [comment]
It should be pretty straightforward to add a button for this. I'll probably make it so that the speed increases from 1x-2.5x by 0.1x every time you tap the button. It's not a great user experience because the button will need to be tapped a bunch of times, but it's a much easier implementation than making some sort of popup.

Or maybe it'll be better in the plugin settings.

I'll work on it in the next couple days. If you have an idea for the UI let me know.

**Comment by serpro69** [comment]
I think a button is fine, so long as you don't need to click through the whole thing up to max to go back to 1x.
I think if you place the value into `audio-note` block, it will be pretty easy to adjust the speed either way, e.g.:

```audio-note
audio: assets/276-paul-grahams-essays-part-2-ads.mp3#t=<start>,<end>&s=1.5
title: Founders Podcast, Episode 276 - Paul Graham's Essays Part 2
transcript: assets/276-paul-grahams-essays-part-2-ads.json
```

**Comment by jjmaldonis** [comment]
Fixed in 34ffd6a. It will be released in v0.1.6.
